chia/types/blockchain_format/proof_of_space.py

```diff
@@ -1,31 +1,20 @@
 from __future__ import annotations
 
 import logging
-from dataclasses import dataclass
 from typing import Optional, cast
 
 from bitstring import BitArray
-from chia_rs import AugSchemeMPL, G1Element, PrivateKey
+from chia_rs import AugSchemeMPL, G1Element, PrivateKey, ProofOfSpace
 from chiapos import Verifier
 
 from chia.consensus.constants import ConsensusConstants
 from chia.types.blockchain_format.sized_bytes import bytes32
 from chia.util.hash import std_hash
-from chia.util.ints import uint8, uint32
-from chia.util.streamable import Streamable, streamable
-
-log = logging.getLogger(__name__)
+from chia.util.ints import uint32
 
+__all__ = ["ProofOfSpace"]
```

Comment:
Note that adding `__all__` actually reduces the default effective `__all__` for `*` imports down to that one thing.  I understand that doing nothing leaves complaints about things not being explicitly reexported.  What about something more like `import chia_rs; ProofOfSpace = chia_rs.ProofOfSpace`?  Does that work?

Reply:
I haven't tried that. I googled a long time ago and people said to do it like this. I'll try